src/styles/typography.css

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Looks good! Only note is that it's not clear what we're doing with `rem`s yet. I believe we're all on board with using them (sure hope so! 😁) but we haven't touched on how to handle larger fonts yet. I'm sure many things will need addressing!